stubs/babel/babel/messages/catalog.pyi

```diff
@@ -1,108 +1,125 @@
-from typing import Any
+import datetime
+from collections import OrderedDict
+from collections.abc import Generator, Iterable, Iterator
+
+from babel.core import Locale
+
+__all__ = ["Message", "Catalog", "TranslationError"]
+
+_MessageID = str | tuple[str, ...] | list[str]
```

Comment:
```suggestion
_MessageID: TypeAlias = str | tuple[str, ...] | list[str]
```

Also needs an import of `TypeAlias` from `typing_extensions`.

Reply:
Thanks!